typescript/syntax/Classes.sdf3

```diff
@@ -0,0 +1,30 @@
+module Classes
+
+imports
+
+  Common
+  Declarations
+  Expressions
+  Functions
+  Names
+  Statements
+  Types
+
+context-free syntax
+
+  ClassDeclaration.ClassDeclaration = <class <BindingIdentifier?> <TypeParameters?> <ClassHeritage?> {
+    <{ClassElement "\n"}*>
```

Comment:
Then we could move `SemiColon` out of `ClassElement` and then do `<SemiColon?>` here